src/test/miniscript_tests.cpp

```diff
@@ -293,7 +293,7 @@ class TestSignatureChecker : public BaseSignatureChecker {
 
 using Fragment = miniscript::Fragment;
 using NodeRef = miniscript::NodeRef<CPubKey>;
-using miniscript::operator"" _mst;
+using miniscript::operator""_mst;
```

Comment:
nit: given the focus on miniscript here, we might as well use the whole namespace like we do for e.g. `std::literals`:
```suggestion
using namespace miniscript;
```